invenio_rdm_records/services/config.py

```diff
@@ -339,6 +339,10 @@ class RDMFileRecordServiceConfig(FileServiceConfig, ConfiguratorMixin):
 
     file_schema = FileSchema
 
+    components = FromConfig(
+        "RDM_FILES_SERVICE_COMPONENTS", default=FileServiceConfig.components
```

Comment:
Can you please add this new config to the `config.py`?

Reply:
i thought we shouldn't do that. there are examples for other component configuration variables which are not added to config.py. as an example [RDM_RECORDS_SERVICE_COMPONENTS](https://github.com/inveniosoftware/invenio-rdm-records/blob/120894386ff7c89870b30cde3a19586e5accb43f/invenio_rdm_records/services/config.py#L472C10-L472C40) is not in [config.py](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py)

Reply:
Maybe simply forgotten?
If it is not in the config.py, it will be quite hard to know that it is possible to override that one. The implementation is [not consistent](https://github.com/inveniosoftware/invenio-app-rdm/blob/3a1b7f2fa86ad887cfb68f30cd0d82c17256dfed/invenio_app_rdm/config.py#L1011) across repos.

Reply:
Fine by me :)